java/view-content-beneath-the-terrain-surface/src/main/java/com/esri/arcgisruntime/sample/viewcontentbeneaththeterrainsurface/MainActivity.java

```diff
@@ -34,6 +34,10 @@
   private static final String TAG = MainActivity.class.getSimpleName();
 
   private SceneView mSceneView;
+  // objects that implement Loadable must be class fields to prevent being garbage collected before loading
+  private Portal mPortal;
+  private PortalItem mPortalItem;
+  private ArcGISScene mScene;
```

Comment:
Let's just use the chainable loading to avoid all these.

Can put error catches in separate listeners where required. Probably just the portal item would suffice?